core/jazz_t-vault/components/util.js

```diff
@@ -0,0 +1,387 @@
+// =========================================================================
```

Comment:
I would rename this to `utils/vault.js`?

Reply:
Done